src/app/component-library/catalog/truncated-short-description/truncated-short-description.component.ts

```diff
@@ -0,0 +1,18 @@
+import {Component, HostBinding, Input, OnInit} from '@angular/core';
+
+@Component({
+  selector: 'truncated-short-description',
+  templateUrl: './truncated-short-description.component.html',
+})
+export class TruncatedShortDescription implements OnInit {
+  @Input() text!: string | undefined;
+  @HostBinding('class.italic') italic = false;
+  @HostBinding('class')
+  get classes() {
+    return 'whitespace-pre-line truncate-lines-5';
+  }
+
+  ngOnInit() {
+    this.italic = !this.text;
```

Comment:
this is problematic. the short descriptions are used in an ngFor, it's possible that the data for this component changes, so only listening to ngOnInit() is not enough.

I did not remark this on the other review, as you can argue that the markdown component only shows when you "spawn" the detail dialog. It's different for the cards, though.

I believe you could do something like this:

```ts
@HostBinding("class.italic")
get italic(): boolean {
   return !this.text;
}
```

If you had "more to calculate" you would have to intercept "ngOnChanges" and have things type unsafe (Angular...)